package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sample",
+  "version": "1.0.0",
+  "description": "![vv-banner.png](./docs/images/vv-call-desktop.png)",
```

Comment:
This is taken by default from README file. Shall i change it to an empty string or some other reasonable description?

Reply:
I am not sure if we want to use "description" in this `package.json`. In my opinion, I think either remove this property or put empty string make sense to me.

I would like to see what others think abou this!

Reply:
Agreed, I think we either omit the description property or change this to another string. The way the string is formatted now is meant to be in markdown file.